libraries/ms-drivers/inc/drv120_relay.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+// Driver for the DRV120 relay
+// Requires GPIO and status to be initialized
+
+#include "gpio.h"
+#include "status.h"
+
+// Storage for GPIO pin to enable relay
+GpioAddress relay_pin;
+
+StatusCode drv120_relay_init(GpioAddress *pin);
```

Comment:
For this function it'd be nice to have a brief explanatory comment about what it does and what `pin` is - it's the enable pin on the DRV120. You should also say that the relay defaults to closed as soon as this function is called.